src/test/java/org/folio/rest/impl/ApiTestBase.java

```diff
@@ -0,0 +1,132 @@
+package org.folio.rest.impl;
+
+import io.restassured.RestAssured;
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.folio.rest.RestVerticle.OKAPI_HEADER_TENANT;
+import static org.folio.rest.RestVerticle.OKAPI_HEADER_TOKEN;
+import static org.folio.rest.impl.ApiTestSuite.mockPort;
```

Comment:
Minor. As far as I remember we started to follow eclipse code style. It doesn't look like static import block is on a right place. Please reformat the imports.